printer_mods/AlexanderT-Moss/270-Clamping-Hinges/README.md

```diff
@@ -1,70 +1,68 @@
+
+
+
+## Update Log ##
+
+**27.10.22** - Overhauled the github page to make stuff just make more sense. Also made slight tweaks to face_plate_x.stl to stop the bolts scratching the frame.
 
 **11.12.21** - Made subtle changes to dimensions to hold panels closer together whilst having them fully seated in the clamps for maximum strength, also changed recommended print settings for stronger face_plate_bottom.stl.
 
- ### Using Foam Tape ###
 
- At the moment I have only tested using 3mm thick foam and that works well, you may be able to get away with 4mm foam, but for anything thicker, you'll need to modify part of the hinge to accommodate this.
- 
- ### Keeping The Doors Shut ###
- 
- I am currently using clamps designed by [_v6cl_](https://github.com/v6cl/My-Voron2.4-Customs/tree/main/Panel_Locker)
 
- ### Different Styles Of Face Plates ###
+ ## Do I Need Foam Tape? ##
+
+ **YES**, You will need to use **3mm foam tape** with these hinges - if you wearn't using foam tape before see this as the right time to add foam tape as you should be using it anyway :)
+
+
+
+ ## How Do I Keep The Doors Shut Without VHB? ##
 
- The file names match up with the digrams below.
+ I am currently using clamps designed by [_v6cl_](https://github.com/v6cl/My-Voron2.4-Customs/tree/main/Panel_Locker) to keep my doors shut (one at the top and one at the bottom). There is also these clamps that are popular by [*Richard M*](https://www.printables.com/model/172368-voron-24-filament-latch-or-any-2020-extrusion) you may wish to use instead but both work just as well.
 
- Face Plate No Logo               |  Face Plate Embossed Logo         | Face Plate Through Logo
- :-------------------------------:|:-------------------------------:|:-------------------------------:
- ![Logo-Variation-0.png](https://github.com/Alexander-T-Moss/VoronUsers/blob/master/printer_mods/AlexanderT-Moss/270-Clamping-Hinges/Images/Logo-Variation-0.png?raw=true)         |  ![Logo-Variation-1.png](https://github.com/Alexander-T-Moss/VoronUsers/blob/master/printer_mods/AlexanderT-Moss/270-Clamping-Hinges/Images/Logo-Variation-1.png?raw=true)  | ![Logo-Variation-2](https://github.com/Alexander-T-Moss/VoronUsers/blob/master/printer_mods/AlexanderT-Moss/270-Clamping-Hinges/Images/Logo-Variation-2.png?raw=true)
+  ## Is The Clamping Force Strong Enough? ##
+
+  If you print these with any half decent filament and your **panels are 3mm thick** then the clamping force is more than sufficiently strong. When printing, the panels are additionaly held in place securly by the door locking clamps linked above. You may find opening the doors quickly will make them shift slightly but the panels can be alligned again by hand quite quickly - I recommend closing the panels together at the same time to help allign them.
```

Comment:
`additionaly` should be `additionally`
`securly` should be `securely`
`alligned` should be `aligned`
`allign` should be `align`